client/extensions/woocommerce/app/orders/index.js

```diff
@@ -8,17 +8,21 @@ import React, { Component } from 'react';
 /**
  * Internal dependencies
  */
+import ActionHeader from 'woocommerce/components/action-header';
+import Button from 'components/button';
+import { getSiteAdminUrl } from 'state/sites/selectors';
 import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
 import Main from 'components/main';
-import OrderHeader from './order-header';
 import OrdersList from './orders-list';
 
 class Orders extends Component {
 	render() {
-		const { className, site, translate } = this.props;
+		const { className, createOrderLink, translate } = this.props;
 		return (
 			<Main className={ className }>
-				<OrderHeader siteSlug={ site ? site.slug : false } breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) } />
+				<ActionHeader breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) }>
```

Comment:
:disappointed: 99 existing translations will be lost with this change.
If the string must change, the following option has already been translated **87** times:
`translate( 'Order:' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).